packages/block-editor/src/components/justify-content-control/ui.js

```diff
@@ -17,12 +17,14 @@ const icons = {
 	'space-between': justifySpaceBetween,
 };
 
-export function JustifyToolbar( {
+function JustifyContentUI( {
 	allowedControls = [ 'left', 'center', 'right', 'space-between' ],
 	isCollapsed = true,
 	onChange,
 	value,
 	popoverProps,
+	isToolbar,
+	isToolbarButton = true,
```

Comment:
Is it necessary here? What would be a use case to set it as `false`? The same question applies to `AlignmentUI`.

Reply:
the reason for this is if this control is rendered outside the block toolbar (which is not the case now), but if we want these controls to be generic, that's something we need to consider.

Reply:
Right, should it consume the toolbar context instead to detect if it should be a toolbar button or not? @diegohaz might have an idea how to do it.

Reply:
I think it's possible to consume the toolbar context, yes. But I'd try a more untangled approach like this:
```jsx
<JustifyContentUI as={ToolbarGroup} />
```
```jsx
<JustifyContentUI as={DropdownMenu} toggleAs={ToolbarButton} />
```
```jsx
<DropdownMenu toggleAs={ToolbarButton} />
```